arangograph/backups.md

```diff
@@ -67,14 +67,38 @@ By default, a backup is not uploaded to the cloud, instead it remains on the
 servers of the deployment. To make a backup that is resilient against server
 (disk) failures, upload the backup to cloud storage. 
 
-When the **Upload backup to cloud storage option** is enabled, the backup is
+When the **Upload backup to cloud storage** option is enabled, the backup is
 preserved for a long time and does not occupy any disk space on the servers.
 This also allows copying the backup to different regions and it can be
 configured in the **Multiple region backup** section.
 
 Uploaded backups are
 required for [cloning](#how-to-clone-deployments-using-backups).
 
+#### Best practices for uploading backups
+
+When utilizing the **Upload backup to cloud storage** feature, a recommended
+approach is to implement a backup strategy that balances granularity and storage
+efficiency.
+
+One effective strategy involves creating a combination of backup intervals and
+retention periods. For instance, consider the following example:
+
+1. Perform 24 hourly backups with a retention period of 24 hours. Hourly backups
```

Comment:
Discussed with @xyll and think we'd prefer the following instead (please wordsmith to make better):

1. Backup every 4 hours, retained for a day
2. Backup every day, retained for a week
3. Backup every week, retained for a month
4. Backup every month, retained for a year

The original hourly backups will likely be too expensive right now, and not having weekly backups is a bit of a gap in the current propsal, so this fixes both issues.

Reply:
Yes, makes sense. Will adjust it. And I think it's great to have these examples in the platform as well!

Reply:
@mattcarabine can you please re-review?